runtime/onert/api/include/nnfw_experimental.h

```diff
@@ -174,6 +174,11 @@ NNFW_STATUS nnfw_pop_pipeline_output(nnfw_session *session, void *outputs);
 //////////////////////////////////////////////
 // Essential APIs for training
 //////////////////////////////////////////////
+typedef enum
+{
+  NNFW_TRAIN_LOSS_MEAN_SQUARED_ERROR = 0,
+  NNFW_TRAIN_LOSS_CATEGORICAL_CROSSENTROPY = 1,
+} NNFW_TRAIN_LOSS;
```

Comment:
```suggestion
} nnfw_train_loss;
```

To be consistent to other enum type name

Reply:
Other enums are the same format.

https://github.com/Samsung/ONE/blob/master/runtime/onert/api/include/nnfw.h#L104
https://github.com/Samsung/ONE/blob/master/runtime/onert/api/include/nnfw.h#L126
https://github.com/Samsung/ONE/blob/master/runtime/onert/api/include/nnfw.h#L145
... :)

Reply:
Ah, you're right. :) I was confused with struct.